docs/rst/maintainer_guide.rst

```diff
@@ -300,6 +300,18 @@ to update the lock file.
 
 .. _sec-maintainer_guide-release:
 
+Continuous integration
+----------------------
+
+Eradiate has a continuous integration scheme built in `Github Actions <https://docs.github.com/en/actions>`_ .
+The action is configured in the ``ci.yml`` file.
+
+Conda setup as well as the compilation of Mitsuba are handled by cmake. Therefore changes to the conda
+environment or the build process do not require any explicit updates to the CI configuration.
```

Comment:
This is a little convoluted, I'd suggest the following (please correct me if I'm wrong):
```suggestion
As per the documented installation process, Conda environment setup is handled using 
the appropriate Makefile and Mitsuba build configuration is done using the CMake preset. 
No CI-specific build setup operations are required.
```